explorer/explorerroutes.go

```diff
@@ -180,19 +180,15 @@ func (exp *explorerUI) TxPage(w http.ResponseWriter, r *http.Request) {
 					tx.TicketInfo.PoolStatus = poolStatus.String()
 				}
 				tx.TicketInfo.SpendStatus = spendStatus.String()
-				tx.TicketInfo.TicketPoolSize = int64(exp.ChainParams.TicketPoolSize)
+				tx.TicketInfo.TicketPoolSize = int64(exp.ChainParams.TicketPoolSize) * int64(exp.ChainParams.TicketsPerBlock)
+				tx.TicketInfo.AveTimeToVote = int64(exp.ChainParams.TicketPoolSize) / int64(exp.ChainParams.TicketsPerBlock)
```

Comment:
My last comment was referring to:
`tx.TicketInfo.TicketPoolSize / int64(exp.ChainParams.TicketsPerBlock)`
As you said, this does reduce to `int64(exp.ChainParams.TicketPoolSize)` because of the odd semantics used in `chaincfg`.

I just though it was important to have `tx.TicketInfo.TicketPoolSize` be 40960, and `tx.TicketInfo.AveTimeToVote` be 8192.

Initially you had `tx.TicketInfo.TicketPoolSize = int64(exp.ChainParams.TicketPoolSize)`, which was confusing because this was 8192, what we wanted to call AveTimeToVote.